README.md

```diff
@@ -1,11 +1,22 @@
-# OpenFisca Web-API
+# OpenFisca Web-API - DEPRECATED
 
-[![Build Status](https://travis-ci.org/openfisca/openfisca-web-api.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-web-api)
+## This package is deprecated
 
-[More build statuses](https://www.openfisca.fr/build-status)
+**The official API is now available in [OpenFisca Core repository](https://github.com/openfisca/openfisca-core).**  
+**The current package isn't maintained anymore: it won't evolve, and it is not guaranteed to be compatible with future versions of OpenFisca.**
+
+The aim of this package was to provide OpenFisca information and computations through a web interface, without installing anything locally.
+
+However, after a few years of existence, it appeared:
+* That versioning OpenFisca API and the Core module apart was difficult to maintain
+* That this API endpoints needed to be improved to match users experiences
+
+## Introduction
+
+[![Build Status](https://travis-ci.org/openfisca/openfisca-web-api.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-web-api) - [More build statuses](https://www.openfisca.fr/build-status)
```

Comment:
We can remove this : for a deprecated package, exposing the build status is not really relevant.